providers/servicenow/connector.go

```diff
@@ -0,0 +1,51 @@
+package servicenow
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/internal/components"
+	"github.com/amp-labs/connectors/internal/components/operations"
+	"github.com/amp-labs/connectors/internal/components/schema"
+	"github.com/amp-labs/connectors/providers"
+)
+
+const (
+	restAPIPrefix = "api"
+)
+
+type Connector struct {
+	// Basic connector
+	*components.Connector
+
+	// Require authenticated client
+	common.RequireAuthenticatedClient
+	// Require workspace
+	common.RequireWorkspace
+	// Require module
+	common.RequireModule
+
+	// Supported operations
+	components.SchemaProvider
+	components.Reader
+	components.Writer
```

Comment:
Please add reader, writer and deleters only when you implement them, because IIRC, they will pass this - 
```
dc, ok := conn.(DeleteConnector)
if ok {
  // servicenow can get here
}
```